crates/biome_js_analyze/tests/specs/suspicious/noCommentText/invalid.tsx.snap

```diff
@@ -4,67 +4,261 @@ expression: invalid.tsx
+invalid.tsx:2:10 lint/suspicious/noCommentText  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Wrap comments inside children within braces.
+  
+    1 │ <>
+  > 2 │     <div>// comment</div>
+      │          ^^^^^^^^^^
+    3 │     <div>/* comment */</div>
+    4 │     <div>/** comment */</div>
+  
+  i Unsafe fix: Wrap the comments with braces
+  
+     1  1 │   <>
+     2    │ - ····<div>//·comment</div>
+        2 │ + ····<div>{/*comment*/}</div>
+     3  3 │       <div>/* comment */</div>
+     4  4 │       <div>/** comment */</div>
+  
+
+```
+
+```
+invalid.tsx:3:10 lint/suspicious/noCommentText  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Wrap comments inside children within braces.
+  
+    1 │ <>
+    2 │     <div>// comment</div>
+  > 3 │     <div>/* comment */</div>
+      │          ^^^^^^^^^^^^^
+    4 │     <div>/** comment */</div>
+    5 │     <div>text /* comment */</div>
+  
+  i Unsafe fix: Wrap the comments with braces
+  
+     1  1 │   <>
+     2  2 │       <div>// comment</div>
+     3    │ - ····<div>/*·comment·*/</div>
+        3 │ + ····<div>{/*comment*/}</div>
+     4  4 │       <div>/** comment */</div>
+     5  5 │       <div>text /* comment */</div>
+  
+
+```
+
+```
+invalid.tsx:4:10 lint/suspicious/noCommentText  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Wrap comments inside children within braces.
+  
+    2 │     <div>// comment</div>
+    3 │     <div>/* comment */</div>
+  > 4 │     <div>/** comment */</div>
+      │          ^^^^^^^^^^^^^^
+    5 │     <div>text /* comment */</div>
+    6 │     <div>/* comment */ text</div>
+  
+  i Unsafe fix: Wrap the comments with braces
+  
+     2  2 │       <div>// comment</div>
+     3  3 │       <div>/* comment */</div>
+     4    │ - ····<div>/**·comment·*/</div>
+        4 │ + ····<div>{/**·comment*/}</div>
+     5  5 │       <div>text /* comment */</div>
+     6  6 │       <div>/* comment */ text</div>
+  
+
+```
+
+```
+invalid.tsx:5:10 lint/suspicious/noCommentText  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
   ! Wrap comments inside children within braces.
   
-  > 1 │ const a3 = <div>// comment</div>;
-      │                 ^^^^^^^^^^
-    2 │ const a4 = <div>/* comment */</div>;
-    3 │ const a5 = <div>/** comment */</div>;
+    3 │     <div>/* comment */</div>
+    4 │     <div>/** comment */</div>
+  > 5 │     <div>text /* comment */</div>
+      │          ^^^^^^^^^^^^^^^^^^
+    6 │     <div>/* comment */ text</div>
+    7 │     <div>
   
   i Unsafe fix: Wrap the comments with braces
   
-    1   │ - const·a3·=·<div>//·comment</div>;
-      1 │ + const·a3·=·<div>{/*·comment*/}</div>;
-    2 2 │   const a4 = <div>/* comment */</div>;
-    3 3 │   const a5 = <div>/** comment */</div>;
+     3  3 │       <div>/* comment */</div>
+     4  4 │       <div>/** comment */</div>
+     5    │ - ····<div>text·/*·comment·*/</div>
+        5 │ + ····<div>text·{/*comment*/}</div>
```

Comment:
The previous suggestion was able to keep the spaces, but now we remove leading and trailing spaces between the word "comment".

Before: `const·a4·=·<div>{/*·comment·*/}</div>;`
Now: `<div>text·{/*comment*/}</div>`

Reply:
@ematipico 
thanks your check, I had missed it...
I fixed in this commit!
[9c6f601](https://github.com/biomejs/biome/pull/3446/commits/9c6f601e8d9fd5134e5a921bb494185495811176)